packages/lesswrong/lib/collections/dialogueMatchPreferences/schema.ts

```diff
@@ -96,7 +96,17 @@ const schema: SchemaType<DbDialogueMatchPreference> = {
     canCreate: ['admins'],
     canRead: ['members', 'admins'],
     canUpdate: ['admins'],
-  }
+  },
+  // deleted: Indicates whether a form has been soft deleted, after a dialogue was published and the form was reset
+  deleted: {
+    type: Boolean,
+    nullable: false,
```

Comment:
With the nullability PRs merged in, `nullable: false` is now provided by default by `schemaDefaultValue`, `arrayOfForeignKeysField`, and `denormalizedCountOfReferences`.  So it doesn't need to be explicitly specified in those cases (unless you want to explicitly make it nullable, in which case you need to put `nullable: true` _after_ `...schemaDefaultValue(...)` or whatever).